jobbergateapi2/config.py

```diff
@@ -3,10 +3,11 @@
 """
 from pydantic import BaseSettings, Field
 
+_DB_RX = r"^(sqlite|postgres)://.+$"
+
 
 class Settings(BaseSettings):
-    DATABASE_URL: str
-    TEST_DATABASE_URL: str
+    DATABASE_URL: str = Field("sqlite:///./sqlite.db?check_same_thread=true", regex=_DB_RX)
     TEST_ENV: bool = Field(False)
     SECRET_KEY: str
```

Comment:
Possibly usage here is [SecretStr](https://pydantic-docs.helpmanual.io/usage/types/#secret-types)